### ISSUE TITLE
Container build can fail when using v1 image format

### ISSUE THREAD

**Issue by pditommaso** [seqeralabs/wave#287]
Container build can fail when the base image used in the Dockefile use the schema v1 format returning the following error/log message 

```
INFO[0000] Retrieving image manifest [quay.io/biocontainers/fastqc:0.11.9--0](http://quay.io/biocontainers/fastqc:0.11.9--0) 
INFO[0000] Retrieving image [quay.io/biocontainers/fastqc:0.11.9--0](http://quay.io/biocontainers/fastqc:0.11.9--0) from registry [quay.io](http://quay.io/) 
ERRO[0001] Error while retrieving image from cache: [quay.io/biocontainers/fastqc:0.11.9--0](http://quay.io/biocontainers/fastqc:0.11.9--0) unsupported MediaType: "application/vnd.docker.distribution.manifest.v1+prettyjws", see https://github.com/google/go-containerregistry/issues/377 
INFO[0001] Retrieving image manifest [quay.io/biocontainers/fastqc:0.11.9--0](http://quay.io/biocontainers/fastqc:0.11.9--0) 
INFO[0001] Retrieving image [quay.io/biocontainers/fastqc:0.11.9--0](http://quay.io/biocontainers/fastqc:0.11.9--0) from registry [quay.io](http://quay.io/) 
error building image: unsupported MediaType: "application/vnd.docker.distribution.manifest.v1+prettyjws", see https://github.com/google/go-containerregistry/issues/377
```

To replicate the error the this oneline dockerfile can be used 

```
FROM quay.io/biocontainers/fastqc:0.11.9--0
```

See related to [this](https://github.com/GoogleContainerTools/kaniko/issues/1836) Kaniko issue.

**Comment by pditommaso** [comment]
This will be solved by #503

**Comment by pditommaso** [comment]
Solved via #503
